sdk/core/azure-core-serializer-avro-apache/src/test/java/com/azure/core/serializer/avro/apache/generatedtestsources/LongLinkedList.java

```diff
@@ -71,8 +71,8 @@ public static LongLinkedList fromByteBuffer(
     return DECODER.decode(b);
   }
 
-  @Deprecated public long value;
-  @Deprecated public com.azure.core.serializer.avro.apache.generatedtestsources.LongLinkedList next;
+   private long value;
```

Comment:
Changes to `azure-core-serializer-avro` and `azure-data-schemaregistry-avro` should be reverted.